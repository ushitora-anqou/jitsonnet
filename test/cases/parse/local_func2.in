local a(v, w=1) = v; a(0)
