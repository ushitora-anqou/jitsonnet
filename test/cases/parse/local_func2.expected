{ Syntax.v =
  (Syntax.Local
     ([(Syntax.BindFunc
          ("a",
           { Syntax.v =
             ([("v", None);
                ("w",
                 (Some { Syntax.v = (Syntax.Number 1.);
                         loc =
                         { Syntax.fname = "test/cases/parse/local_func2.in";
                           ran =
                           (Some { Syntax.startpos =
                                   { Syntax.line = 1; column = 13 };
                                   endpos = { Syntax.line = 1; column = 14 }
                                   })
                           }
                         }))
                ],
              { Syntax.v = (Syntax.Var "v");
                loc =
                { Syntax.fname = "test/cases/parse/local_func2.in";
                  ran =
                  (Some { Syntax.startpos = { Syntax.line = 1; column = 18 };
                          endpos = { Syntax.line = 1; column = 19 } })
                  }
                });
             loc =
             { Syntax.fname = "test/cases/parse/local_func2.in";
               ran =
               (Some { Syntax.startpos = { Syntax.line = 1; column = 6 };
                       endpos = { Syntax.line = 1; column = 19 } })
               }
             }))
        ],
      { Syntax.v =
        (Syntax.Call
           ({ Syntax.v = (Syntax.Var "a");
              loc =
              { Syntax.fname = "test/cases/parse/local_func2.in";
                ran =
                (Some { Syntax.startpos = { Syntax.line = 1; column = 21 };
                        endpos = { Syntax.line = 1; column = 22 } })
                }
              },
            ([{ Syntax.v = (Syntax.Number 0.);
                loc =
                { Syntax.fname = "test/cases/parse/local_func2.in";
                  ran =
                  (Some { Syntax.startpos = { Syntax.line = 1; column = 23 };
                          endpos = { Syntax.line = 1; column = 24 } })
                  }
                }
               ],
             []),
            false));
        loc =
        { Syntax.fname = "test/cases/parse/local_func2.in";
          ran =
          (Some { Syntax.startpos = { Syntax.line = 1; column = 21 };
                  endpos = { Syntax.line = 1; column = 25 } })
          }
        }));
  loc =
  { Syntax.fname = "test/cases/parse/local_func2.in";
    ran =
    (Some { Syntax.startpos = { Syntax.line = 1; column = 0 };
            endpos = { Syntax.line = 1; column = 25 } })
    }
  }